src/theme/colors.js

```diff
@@ -156,7 +156,7 @@ const colors = {
     textSoft: '#A9A9A9',
     textSoftInverted: '#7D7D7D',
     disabled: '#656565',
-    accentColorBriefing: '#07809A',
+    accentColorBriefing: '#0A99B8',
```

Comment:
You'll need to add a format mapping for `#07809a` and `#07809A`  to `accentColorBriefing`. We'll eventually change publikator to only allow selecting color names.

Reply:
done.